examples/kafka-broker/verify.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -e
+
+export NAME=kafka-broker
+
+# shellcheck source=examples/verify-common.sh
+. "$(dirname "${BASH_SOURCE[0]}")/../verify-common.sh"
+
+run_log "Waiting for Kafka on 9092."
+while true; do if docker-compose logs -f kafka | grep -q 'started (kafka.server.KafkaServer)'; then
```

Comment:
It may be failed in CI, could you replace with:
```
while true; do
   if xxx
   else
   fi
done

Reply:
Updated, please help verify again